modules/aleph/aleph.module

```diff
@@ -86,6 +86,12 @@ function aleph_ding_provider() {
         'prefix' => 'debt',
         'file' => 'includes/aleph.debt.inc',
       ),
+      'provides' => array(
+        'payment' => array(
+          'prefix' => 'payment',
+          'file' => 'includes/dibs.payment.inc',
```

Comment:
Filename `dibs.payment.inc`? That does not seem right.

Reply:
Ignore this, it was a thinko, I'll rewrite it out of history...